reports/final/report.tex

```diff
@@ -57,16 +57,16 @@
 \pagenumbering{gobble}
 \section*{Executive Summary}
 % ERIC OWNS THIS SECTION
-The Covid-19 pandemic abruptly forced many educational institutions towards online
```

Comment:
Did you mean to edit the exec summary or do we need to rebase?

Reply:
these edits were intentional